runtime/interpreter/account.go

```diff
@@ -33,6 +33,7 @@ var authAccountFieldNames = []string{
 	sema.AuthAccountAddressField,
 	sema.AuthAccountContractsField,
 	sema.AuthAccountKeysField,
+	sema.AuthAccountInboxField,
```

Comment:
Confused by this; what effect does this actually have given that none of the interpreter tests failed without this? I would have expected not to be able to access the `inbox` field without specifying this.

Reply:
See the usage of `FieldNames` in `SimpleCompositeValue`. For example, they are used to iterate over the fields, prepare a string representation, check static type conformance, etc